src/components/QueryOutput/QueryOutput.js

```diff
@@ -1976,9 +1978,12 @@ export class QueryOutput extends React.Component {
   renderFooter = () => {
     const shouldRenderRT = this.shouldRenderReverseTranslation()
     const shouldRenderDLW = this.shouldRenderDataLimitWarning()
-
+    const footerClassName =
```

Comment:
This seems a bit confusing to me. What if we did something like this

```
const footerClassName =
    `query-output-footer-top
    ${!shouldRenderRT ? 'no-margin' : ''}
    ${this.props.reverseTranslationPlacement}`
```

Then the css selector could just be

`.query-output-footer.top` or `.query-output-footer.bottom`